parachain/runtime/interlay/src/lib.rs

```diff
@@ -866,7 +874,7 @@ impl annuity::Config<VaultAnnuityInstance> for Runtime {
     type BlockNumberToBalance = BlockNumberToBalance;
     type EmissionPeriod = EmissionPeriod;
     type TotalWrapped = TotalWrapped;
-    type WeightInfo = ();
+    type WeightInfo = weights::annuity_vault_annuity::WeightInfo<Runtime>;
```

Comment:
If we remove the `call`I think this is also not necessary

Reply:
We do need weights for the root extrinsics and the `on_initialize` hook.